### ISSUE TITLE
Backend error

### ISSUE THREAD

**Issue by LemonLight** [PEXPlugins/PermissionsEx#2577]
Hello, 

I have got a big error in my logs. Any idea?

Thanks !

Here there is:

[10:46:29] [pool-16-thread-1/ERROR]: [PermissionsEx] Error occurred with PermissionsEx! Please post it to http://git.io/<html><body>You are being <a href="https://git.io/create">redirected</a>.</body></html>. Full error:
ru.tehkode.permissions.exceptions.PermissionBackendException: com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'prefix' in 'field list'
	at ru.tehkode.permissions.backends.sql.SQLBackend$3.performUpdate(SQLBackend.java:186)
	at ru.tehkode.permissions.backends.PermissionBackend.performSchemaUpdate(PermissionBackend.java:90)
	at ru.tehkode.permissions.backends.sql.SQLBackend.<init>(SQLBackend.java:192)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at ru.tehkode.permissions.backends.PermissionBackend.getBackend(PermissionBackend.java:392)
	at ru.tehkode.permissions.backends.PermissionBackend.getBackend(PermissionBackend.java:367)
	at ru.tehkode.permissions.PermissionManager.createBackend(PermissionManager.java:624)
	at ru.tehkode.permissions.PermissionManager.setBackend(PermissionManager.java:606)
	at ru.tehkode.permissions.PermissionManager.initBackend(PermissionManager.java:693)
	at ru.tehkode.permissions.PermissionManager.<init>(PermissionManager.java:67)
	at ru.tehkode.permissions.bukkit.PermissionsEx.onEnable(PermissionsEx.java:175)
	at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:292)
	at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:337)
	at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:405)
	at org.bukkit.craftbukkit.v1_10_R1.CraftServer.enablePlugin(CraftServer.java:373)
	at org.bukkit.craftbukkit.v1_10_R1.CraftServer.enablePlugins(CraftServer.java:323)
	at net.minecraft.server.v1_10_R1.MinecraftServer.t(MinecraftServer.java:412)
	at net.minecraft.server.v1_10_R1.MinecraftServer.l(MinecraftServer.java:377)
	at net.minecraft.server.v1_10_R1.MinecraftServer.a(MinecraftServer.java:332)
	at net.minecraft.server.v1_10_R1.DedicatedServer.init(DedicatedServer.java:271)
	at net.minecraft.server.v1_10_R1.MinecraftServer.run(MinecraftServer.java:535)
	at java.lang.Thread.run(Thread.java:745)
Caused by: com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'prefix' in 'field list'
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:404)
	at com.mysql.jdbc.Util.getInstance(Util.java:387)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:941)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3870)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3806)
	at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:2470)
	at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:2617)
	at com.mysql.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2550)
	at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1861)
	at com.mysql.jdbc.PreparedStatement.executeQuery(PreparedStatement.java:1962)
	at ru.tehkode.libs.org.apache.commons.dbcp.DelegatingPreparedStatement.executeQuery(DelegatingPreparedStatement.java:96)
	at ru.tehkode.libs.org.apache.commons.dbcp.DelegatingPreparedStatement.executeQuery(DelegatingPreparedStatement.java:96)
	at ru.tehkode.permissions.backends.sql.SQLBackend$3.performUpdate(SQLBackend.java:161)
	... 24 more

**Comment by Faxed** [comment]
Without meaning to commandeer this issue, I'm also having the exact same issue. Going through this Github, it appears it has been reported as an issue over sixty times in the past year. It has slowly driven me to my wit's end, especially as I had it working once and then cleared it. 

So far as I can see, it's incredibly temperamental when it comes to MySQL. However, to fix this issue I found that I had to drop every table in the database and start from scratch - If any were remaining, when importing from another source, it lead to a 'schema_version' entry not being made. 

So, if you are having this issue when attempting to switch backends, I'd suggest dropping all of the tables (rather than emptying them) in the database (dealing with PEX) and then trying to import again.
